src/main/java/com/minecolonies/coremod/proxy/ClientProxy.java

```diff
@@ -20,6 +20,7 @@
 import com.minecolonies.coremod.tileentities.ScarecrowTileEntity;
 import com.minecolonies.coremod.tileentities.TileEntityColonyBuilding;
 import com.minecolonies.coremod.tileentities.TileEntityInfoPoster;
+import com.minecolonies.structures.client.TemplateBlockAccessTransformHandler;
 import com.minecolonies.structures.client.TemplateRenderHandler;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.structures.client.TemplateRenderHandler'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)